jmp-cloud-service-impl/src/test/java/com/epam/jmp/impl/ServiceImplTest.java

```diff
@@ -0,0 +1,139 @@
+package com.epam.jmp.impl;
+
+import com.epam.jmp.dto.BankCard;
+import com.epam.jmp.dto.Subscription;
+import com.epam.jmp.dto.User;
+import com.epam.jmp.repo.SubscriptionRepository;
+import com.epam.jmp.repo.UserRepository;
+import com.epam.jmp.repo.impl.SubscriptionNotFoundException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.*;
```

Comment:
I wouldn't import the whole packages - only those classes that you need - but some may argue with that approach - so I'll leave the decision to you.